src/ui/server/helpers/sanitise-data/sanitise-value/index.ts

```diff
@@ -0,0 +1,129 @@
+import { SanitiseValueObjParams } from '../../../../types';
+
+const {
+  ACCOUNT: { SECURITY_CODE },
+  COMPANIES_HOUSE: { COMPANY_NUMBER, COMPANY_SIC },
+  EXPORTER_BUSINESS: {
+    YOUR_COMPANY: { PHONE_NUMBER },
+    NATURE_OF_YOUR_BUSINESS: { GOODS_OR_SERVICES },
+    BROKER: { ADDRESS_LINE_1, ADDRESS_LINE_2, POSTCODE, NAME: BROKER_NAME },
+  },
+  POLICY: {
+    CONTRACT_POLICY: { CREDIT_PERIOD_WITH_BUYER },
+    ABOUT_GOODS_OR_SERVICES: { DESCRIPTION },
+  },
+  YOUR_BUYER: {
+    COMPANY_OR_ORGANISATION: { NAME, REGISTRATION_NUMBER, ADDRESS, FIRST_NAME, LAST_NAME, POSITION, WEBSITE },
+  },
+} = INSURANCE_FIELD_IDS;
+
+/**
+ * STRING_NUMBER_FIELDS
+ * Add all string fields here that could be numbers
+ * Explicit exemptions list of field IDs in the insurance forms that are string fields that could be submitted as a pure number.
+ * If one of these fields is pure numbers, our default "sanitise data" behaviour is to transform a string number into a number type.
+ * However, we do not want this to happen with these fields. This means that:
+ * - The data is saved correctly
+ * - We avoid adding additional error handling in the UI.
+ * - We avoid a "problem with service" page scenario where the data save fails, because it tries to save a number type as a text string type.
+ * @returns {Array} Field IDs
+ */
+export const STRING_NUMBER_FIELDS = [
+  SECURITY_CODE,
+  CREDIT_PERIOD_WITH_BUYER,
+  DESCRIPTION,
+  COMPANY_NUMBER,
+  COMPANY_SIC,
+  PHONE_NUMBER,
+  GOODS_OR_SERVICES,
+  ADDRESS_LINE_1,
+  ADDRESS_LINE_2,
+  POSTCODE,
+  BROKER_NAME,
+  NAME,
+  REGISTRATION_NUMBER,
+  ADDRESS,
+  FIRST_NAME,
+  LAST_NAME,
+  POSITION,
+];
+
+/**
+ * replaceCharactersWithCharacterCode
+ * Replace certain characters with character codes
+ * @param {String} Field value
+ * @returns {String}
+ */
+export const replaceCharactersWithCharacterCode = (str: string) =>
+  str
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#x27;')
+    .replace(/\//g, '&#x2F;')
+    .replace(/\*/g, '&#42;');
+
+/**
+ * shouldChangeToNumber
+ * Check if a form field value should change from a string to a number
+ * @param {Object} Field key and value
+ * @returns {Boolean}
+ */
+export const shouldChangeToNumber = ({ key, value }: SanitiseValueObjParams) => {
+  if (key) {
+    if (STRING_NUMBER_FIELDS.includes(key) || isEmptyString(String(value))) {
+      return false;
+    }
+  }
+
+  if (typeof value === 'string') {
+    const stripped = stripCommas(value);
+
+    if (isNumber(stripped)) {
+      return true;
+    }
+  }
+
+  return false;
+};
+
+/**
+ * sanitiseValue
+ * Sanitise a form field value
+ * @param {Object} Field key and value
+ * @returns {Boolean}
+ */
+export const sanitiseValue = ({ key, value }: SanitiseValueObjParams) => {
+  if (value === 'true' || value === true) {
+    return true;
+  }
+
+  if (value === 'false' || value === false) {
```

Comment:
would this be used anywhere else? does it need a helper?

Reply:
I don't believe so - I think we're OK to keep as it is - this has been in place for a long time, just moved the file/function in this PR. Happy to extract this if required though.